src/elements/content-sidebar/activity-feed/activity-feed/ActivityThread.js

```diff
@@ -53,7 +56,9 @@ const ActivityThread = ({
     hasReplies,
     isAlwaysExpanded = false,
     isRepliesLoading,
+    isThreadSelected = false,
     mentionSelectorContacts,
+    onCommentSelect,
```

Comment:
All of the other callbacks use `onReply`. Can we rename this to `onReplySelect` for consistency?